src/art/local/api.py

```diff
@@ -273,12 +273,14 @@ async def _train_model(
             )
             return
         disk_packed_tensors = packed_tensors_to_dir(
-            packed_tensors, f"{get_output_dir_from_model(model)}/tensors"
+            packed_tensors, f"{get_model_dir(self._path, model)}/tensors"
         )
         results: list[dict[str, float]] = []
-        pbar = tqdm.tqdm(total=disk_packed_tensors["num_sequences"], desc="train")
-        async for result in service.train(disk_packed_tensors, config, _config):
+        num_steps = disk_packed_tensors["num_sequences"]
```

Comment:
Is this a different kind of step than the step that refers to a complete iteration through the training loop?

Reply:
Yeah, these are gradient steps. We could rename this variable to `num_grad(ient)_steps` to clarify.

Reply:
Will do!